### ISSUE TITLE
Does "io.quarkiverse.jpastreamer:quarkus-jpastreamer:1.0.1" contain everything or do i need to add the annotation processor separately?

### ISSUE THREAD

**Issue by BenjaminJacober** [speedment/jpa-streamer#295]
When using the quarkiverse dependency, do i still need to add the annotationProcessor line in gradle or not? JPAStreamer seems to work but it's terribly slow and according to https://speedment.github.io/jpa-streamer/jpa-streamer/1.1.2/predicates/predicates.html#_fields i should really use the classes with the $ name for it to be faster. The problem is, they're nonexistent after rebuilding, I can't find them in any build folder.

**Comment by julgus** [comment]
Yes, we strongly recommend using the generated fields, as JPAStreamer cannot optimize the pipeline when using regular lambdas. 

I must admit I do not have much first hand experience with Gradle and Quarkus, but I would suggest trying to add the annotation processor line as described in: 
https://speedment.github.io/jpa-streamer/jpa-streamer/1.1.2/get-jpa-streamer/install-gradle.html.

If that does not work, could you please share some more information about your project so I can try reproducing the issue?

**Comment by julgus** [comment]
Thank you for taking the time to detail your issue. How strange that IntelliJ will not display the $ classes. I just tried setting up a Gradle project myself and the generated entities are placed in build/classes/generated/sources/annotationProcessor. 
I am specifying the annotationProcessor in the build-file however, have you tried if that makes a difference? 

![image](https://user-images.githubusercontent.com/26650460/219676425-3367b8aa-6eb3-45ee-a41c-91e3a270355d.png)

To try to sort this up, I have some follow-up questions: 

1. While strange, IntelliJ sometimes gets confused for different reasons. Have you tried invalidating the IntelliJ caches? 
2. Are all Entity classes placed in the same package? Is there any apparent difference between entities that do trigger the annotation processor and the ones that do not? Are you by any chance using Panache and the Active Record pattern? 
3. It may not recognize them if they are placed in a location that is not accepted as a source file, which could be the case if they are placed in /out.
4. This sounds very strange, would you mind sharing one of the original files and the output so I can try and reproduce it? Specifically, your observation about Enums is interesting. 
5. Whether or not your Repositories use JPAStreamer should not impact if files are generated for that entity. $-classes should be generated for all classes annotated with @Entity. 
6. Could you share an example and the matching generated class? I need to investigate this further.

**Comment by BenjaminJacober** [comment]
Ah, sorry i should've mentioned that as well.
0. I just realised that the generated files are in fact not the ones from JPAStreamer. If i go onto our dev branch without the JPAStreamer stuff and do a fresh compile they are also there. Probably from a different dep (maybe Lombok).

1. Yep, already did that.
2. a. No, we even have separate projects with their own src and main folders. I put the dependency a level above all these projects because we want to use them in each of them. Maybe i should try to put it on a lower level?
2. b. - (see 0.) We don't use Panache.
3. Hmm. What could be a reason for that? They're in src/main/.... (The /out folder only exists on the JPAStreamer branch, so I'm assuming it's got something to do with that. It's completely empty besides an empty beans.xml file)
4.  I can't it's company code, (see 0.)
5. Ok, thanks for clearing that up.
6. - (see 0.)

**Comment by BenjaminJacober** [comment]
Is there any way that i can check if the annotation processor is even working? The JPAStreamer itself i can see in the logs and also in my code as working but the field generator seems to be nonexistent.

**Comment by BenjaminJacober** [comment]
Note: JPA Streamer Field Generator Processor
warning: Supported source version 'RELEASE_8' from annotation processor 'org.gra
dle.api.internal.tasks.compile.processing.TimeTrackingProcessor' less than -sour
ce '17'
1 warning

> Task :xxx:compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':xxx:compileJava'.
> java.lang.NoClassDefFoundError: javax/persistence/Entity

Does this help? I think I've gotten a bit further
